plugins/rtmp-services/data/locale/en-US.ini

```diff
@@ -3,8 +3,12 @@ CustomStreamingServer="Custom Streaming Server"
 Service="Service"
 Server="Server"
 Server.Auto="Auto (Recommended)"
+Server.AutoRTMPS="Auto (RTMPS, Recommended)"
+Server.AutoRTMP="Auto (RTMP)"
 StreamKey="Stream key"
 UseAuth="Use authentication"
 Username="Username"
 Password="Password"
 ShowAll="Show all services"
+MultitrackVideo.Disclaimer="%1 automatically optimizes your settings to encode and send multiple video qualities to %2, powering the best possible viewer experience. Selecting this option will send %2 information about your computer and software setup."
```

Comment:
Please remove `powering the best possible viewer experience`. We try to avoid 'marketing' tone within the app

Reply:
Just clarifying that this string is added in #10543, not here, so this change and feedback need to be done there.

Reply:
Just closing the loop on this one that https://github.com/obsproject/obs-studio/pull/10543 included the verbiage fix, so I think this thread is resolved.

Reply:
Resolving.